index.html

```diff
@@ -47,12 +47,324 @@ <h1 class="heading">Hey there. I’m Sean John.<br /> <span id="green-text">I’
+                                </clipPath>
+                                </defs>
+                                </svg>
+                                
+                        </div>
+                        <div class="quality-content">
+                            <div class="quality-name">Skills</div>
+                            <ul class="skill-tags">
+                                <li class="skill-tag">Codekit</li>
+                                <li class="skill-tag">GitHub</li>
+                                <li class="skill-tag">Codepen</li>
+                                <li class="skill-tag">Codepen</li>
+                                <li class="skill-tag">Codepen</li>
+                            </ul>
+                        </div>
+                        
+
+
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <div class="section contact-section">
+            <div class="container contact">
+                <div class="contact-info">
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </div>
+
+                <div class="contact-form">
+                    <form class="form-block" action="https://formspree.io/f/xeqnpgqv" method="post">
+                        <input placeholder="Full name" type="text" maxlength="30" required>
+                        <input placeholder="Email Address" type="email" name="email" id="email" required>
+                        <textarea placeholder="Message" name="textarea" id="textarea" cols="30" rows="6" required maxlength="500"></textarea>
+                        <div class="button-group">
+                            <button class="see-project" type="submit">Get in touch</button>
+                        </div>
+                    </form>
+                </div>
+            </div>
+        </div>
     </main>
+    <footer class="footer-section">
+        <ul class="icons">
+            <li class="social-item"><a class="social-icon" href="https://github.com/johnalpha74/"><img src="/assets/icons/github.svg" alt="social media icons"></a></li>
+            <li class="social-item"> <a class="social-icon" href="https://www.linkedin.com/in/johnalpha74/"><img src="/assets/icons/linkedin.svg" alt="social media icons"></a></li>
+            <li class="social-item"><a class="social-icon" href="#"><img src="/assets/icons/hey.svg" alt="social media icons"></a></li>
+            <li class="social-item"><a class="social-icon" href="https://twitter.com/theagilesean"><img src="/assets/icons/twitter.svg" alt="social media icons"></a></li>
+            <li class="social-item"><a class="social-icon" href="#"><img src="/assets/icons/medium.svg" alt="social media icons"></a></li>
+        </ul>
+        <a class="email-link" href="mailto:email@example.com">myemailhere@email.com</a>
+    </footer>
```

Comment:
- [ ] Please consider fixing the design of the form on the mobile view to match the Figma design.
![image](https://user-images.githubusercontent.com/62290545/165592170-2ece77b2-b8dc-4588-8a33-0698799db0cb.png)